packages/core/src/utils/breakpoints.ts

```diff
@@ -109,16 +109,18 @@ export const getValueForBreakpoint = (
   if (valuesByBreakpoint instanceof Object) {
     // Assume that the values are sorted by media query to apply the cascading CSS logic
     for (let index = activeBreakpointIndex; index >= 0; index--) {
-      const breakpointId = breakpoints[index].id;
+      const breakpointId = breakpoints[index]?.id;
       if (valuesByBreakpoint[breakpointId]) {
         // If the value is defined, we use it and stop the breakpoints cascade
         return eventuallyResolveDesignTokens(valuesByBreakpoint[breakpointId]);
       }
     }
     // If no breakpoint matched, we search and apply the fallback breakpoint
     const fallbackBreakpointIndex = getFallbackBreakpointIndex(breakpoints);
-    const fallbackBreakpointId = breakpoints[fallbackBreakpointIndex].id;
-    return eventuallyResolveDesignTokens(valuesByBreakpoint[fallbackBreakpointId]);
+    const fallbackBreakpointId = breakpoints[fallbackBreakpointIndex]?.id;
```

Comment:
This is probably never going to fail as long as there is at least one breakpoint available but doesn't hurt to be a bit more defensive here